code/modules/food_and_drinks/kitchen_machinery/microwave.dm

```diff
@@ -255,7 +255,7 @@
 		break
```

Comment:
Poor use of the break statement

Reply:
I mean it's a little silly, but it's ok. the min doesn't need to be there, and the fact it is makes me think it's aged 20mmmmm13? code?
![image](https://user-images.githubusercontent.com/58055496/91771119-447f3d80-eb97-11ea-8faf-790f98a48087.png)

Reply:
the fact ryll didn't fix this speaks a lot to our society

Reply:
it's also not even their code, so even more EH